projects/1.19/assets/quark/quark/lang/zh_cn.json

```
@@ -1,1296 +1,3379 @@
+
+	"quark.config.building.vertical_slabs.name": "竖直台阶",
+	"quark.config.building.vertical_slabs.desc": "将3个台阶摆成一列，就能制作3个对应的竖直台阶。\n每个有台阶的原版/夸克方块都存在竖直台阶变种。",
+	"quark.config.building.vertical_slabs.allow_side_connections.name": "允许侧面连接",
+	"quark.config.building.vertical_slabs.allow_side_connections.desc": "墙和玻璃板是否尝试与竖直台阶的侧面连接？",
+
+	"quark.config.building.wooden_posts.name": "木杆",
+	"quark.config.building.wooden_posts.desc": "木杆可以用原木或剥皮原木制作。\n你可以垂直/水平地铺设这些细长原木样式的木方块。它们也可以连接到锁链或灯笼。",
+
+
+
+	"quark.config.management.automatic_tool_restock.name": "工具自动补给",
+	"quark.config.management.automatic_tool_restock.desc": "你快捷栏上的工具损坏时，此项会自动补给，从你的物品栏中取出同类型的工具替换上去。\n它会优先考虑附有类似魔咒的工具，例如精准采集。",
+	"quark.config.management.automatic_tool_restock.check_hotbar.name": "检查快捷栏",
+	"quark.config.management.automatic_tool_restock.check_hotbar.desc": "允许将快捷栏的物品拉到另一个栏位",
+	"quark.config.management.automatic_tool_restock.enable_enchant_matching.name": "启用魔咒匹配",
+	"quark.config.management.automatic_tool_restock.enable_enchant_matching.desc": "允许比较魔咒来找到替代品",
+	"quark.config.management.automatic_tool_restock.enable_loose_matching.name": "启用宽松匹配",
+	"quark.config.management.automatic_tool_restock.enable_loose_matching.desc": "允许使用同类型的不同工具替换你用坏的工具",
+	"quark.config.management.automatic_tool_restock.ignore_anti_overlap.name": "无视反重叠系统",
+	"quark.config.management.automatic_tool_restock.ignore_anti_overlap.desc": "若加载了下列模组，此功能会自行禁用：\n - inventorytweaks（R键整理）\n这是用来防止内容重叠的。\n打开此项后，即使加载了上述模组也会强制加载此功能。",
+	"quark.config.management.automatic_tool_restock.ignored_items.name": "无视的物品",
+	"quark.config.management.automatic_tool_restock.ignored_items.desc": "补给功能将忽略此列表中的任何物品",
+	"quark.config.management.automatic_tool_restock.important_enchantments.name": "重要的魔咒",
+	"quark.config.management.automatic_tool_restock.important_enchantments.desc": "被认为很重要的魔咒，在寻找替代品时具有特别优先权",
+	"quark.config.management.automatic_tool_restock.unstackables_only.name": "仅限于无法堆叠的物品",
+
+	"quark.config.management.easy_transfering.name": "简单物品移动",
+	"quark.config.management.easy_transfering.desc": "箱子中添加了两个新按钮：「存入」与「取出」。\n「存入」按钮将尝试将物品栏（不包含快捷栏）中的所有物品存入箱子。\n「取出」按钮将尝试将箱子中的所有物品拉入物品栏。\n按住Shift时按下这些按钮，它们只会移动目标侧已有的物品。（例如，箱子里有石头和泥土，物品栏中有石头和沙子，则按住Shift时按下「存入」按钮只会移动石头）\n1.16.4+：还添加了一个新按钮，可切换Shift键的锁定状态。对于肌肉有问题的人来说很有用！",
+	"quark.config.management.easy_transfering.enable_shift_lock.name": "启用Shift锁定按钮",
+
+	"quark.config.management.expanded_item_interactions.name": "物品交互扩展",
+	"quark.config.management.expanded_item_interactions.desc": "添加了一些在物品栏中与物品交互的新方法。\n(1) 用光标拖着某个物品（不是潜影盒）时，你可以右击未满的潜影盒，将物品收入其中。或者，你也可以拖着潜影盒右击该物品。\n(2) 右击物品栏中的盔甲物品，会自动穿戴它或是替换你正穿戴的盔甲。\n(3) 现在可以拿着物品对熔岩桶右击，以永久摧毁它们。这种方式无法摧毁潜影盒和免疫火焰伤害的物品（如下界合金）。\n在物品栏中，可以右击打开潜影盒。",
+	"quark.config.management.expanded_item_interactions.allow_opening_shulker_boxes.name": "允许打开潜影盒",
+	"quark.config.management.expanded_item_interactions.allow_opening_shulker_boxes.desc": "在物品栏中，可以右击打开潜影盒。",
+	"quark.config.management.expanded_item_interactions.allow_rotating_bundles.name": "允许旋转收纳袋",
```

```suggestion
	"quark.config.management.expanded_item_interactions.allow_rotating_bundles.name": "允许循环收纳袋槽位",
```
没有原文确实不大方便，乍一看还以为是旋转收纳袋的材质

---

`没有原文` 是什么意思？难道你现在都没有看原文，是直接在游戏里测试的吗？虽然这个 `原文` 确实很分散就是了……
`.name` 直接看key名就完事，`.desc` 的话
- 部分来自源码中的硬编码，通常可以通过module名（比如 `ExpandedItemInteractionsModule.java`）找到，一般打到第二个大写字母就能找到了
- 其他绝大部分如一楼所说，来自[英文wiki](https://quarkmod.net/#features)
- 可能还剩下极小部分我自己加的还没有删除的部分（大部分这些译注都在[去除译注](https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/pull/3326/commits/0ce9211ac232e155a0a9bacafbed866ca7dfb05f)这个提交中移除了，懒得找漏网之鱼）